o2a/converter/parsed_action_node.py

```diff
@@ -27,8 +27,8 @@
 class ParsedActionNode:
     """Class for parsed Oozie workflow node"""
 
-    def __init__(self, mapper: BaseMapper, name=None, tasks=None, relations=None):
-        self.mapper = mapper
+    def __init__(self, mapper=None, name=None, tasks=None, relations=None):
+        self.mapper: BaseMapper = mapper
```

Comment:
I'm not sure I like the approach of having nodes without a mapper. This creates a leaky abstraction and the problems that I outlined in the comments above.

Reply:
I introduced a different solution.